src/main/java/aegis/Aegis.java

```diff
@@ -0,0 +1,282 @@
+
+public class Aegis {
+
+    private final static String AEGIS_LOGO = """
+                                                                            \s
+                        **                                                  \s
+                     *****                                  *               \s
+                    *  ***                                 ***              \s
+                       ***                                  *               \s
+                      *  **                                          ****   \s
+                      *  **          ***        ****      ***       * **** *\s
+                     *    **        * ***      *  ***  *   ***     **  **** \s
+                     *    **       *   ***    *    ****     **    ****      \s
+                    *      **     **    ***  **     **      **      ***     \s
+                    *********     ********   **     **      **        ***   \s
+                   *        **    *******    **     **      **          *** \s
+                   *        **    **         **     **      **     ****  ** \s
+                  *****      **   ****    *  **     **      **    * **** *  \s
+                 *   ****    ** *  *******    ********      *** *    ****   \s
+                *     **      **    *****       *** ***      ***            \s
+                *                                    ***                    \s
+                 **                            ****   ***                   \s
+                                             *******  **                    \s
+                                            *     ****                      \s
+                                                                            \s""";
+
+    private final static String SEPARATOR = "--------------------------------------------------";
+    private final static String FILE_PATH = "./data/duke.txt";
+
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+
+        System.out.println(SEPARATOR);
+        System.out.println("Hello, this is\n" + AEGIS_LOGO);
+
+        System.out.println(SEPARATOR);
+        System.out.println("Hello! This is Aegis, an Anti-Shadow Suppression Weapon and a chatbot.");
+        System.out.println("What can I do for you?");
+
+        ArrayList<Task> taskList = new ArrayList<>();
+        loadTasksFromFile(taskList);
+
+        while (true) {
+            System.out.println();
+            System.out.println("---------------BEGIN--OF--PROMPT------------------");
+
+            String input = scanner.nextLine().trim();
+
+            if (input.equalsIgnoreCase("bye")) {
+                break;
+            }
+
+            handleCommand(input, taskList);
+        }
+
+        System.out.println(" Bye. Hope to see you again soon!");
+        System.out.println();
+        System.out.println("-------------END--OF--CONVERSATION----------------");
+
+        scanner.close();
```

Comment:
You may consider extracting greeting part and processing command part into separate methods as well for the same abstraction level.